alertproduction.tex

```diff
@@ -25,20 +37,25 @@ \section{Alert Production in Commissioning and Early Operations}
 Rubin aims to scale up alert production during commissioning with the aim of beginning early Alert Production over a progressively increasing fraction of the sky as soon as is feasible following Rubin First Light  (\S~\ref{sec:timeline}).
 \citeds{RTN-061} describes the criteria for sending the first Rubin alerts.
 Once begun, Alert Production will then proceed continuously into the full LSST survey.
-Alerts generated during the early science period may be produced with higher latency, and access to images and the PPDB may not be available during this phase.
+Alerts generated during the early science period may be produced with higher latency, and access to images and the PPDB (\S~\ref{sec:ppdb}) may not be available during this phase.
 During  commissioning and early Operations periods,  alert packets for moving objects might not include the associated historical source records , and parameters such as phase curve slope (G) would be empty until sufficient detections exist to derive them.
-% Table~\ref{tab:prompt-data-products} lists the various alert and prompt processing data products currently planned at each phase of alert production during commissioning and the first two years of the LSST survey.
-%Phase 1 covers the commissioning period and phase 2 covers early survey operations.
-During routine LSST operations, prompt image data products, including  raw images, processed single visit images (PVIs), difference images, and template images, will be made available no earlier than 80 hours following camera readout.
-During the first 6 months of the LSST,  prompt PVIs and difference images may be released with higher latency as Rubin continues to understand data quality and scale up services.
 
-Further details on Alert Production in Commissioning and Early Operations will be provided in the coming months.
+The first Rubin alerts are currently expected to begin between December 2025 and February 2026. 
+
+%%% Prompt images
+\subsection{Prompt Images}
+\label{sec:prompt-images}
+
+Starting in early operations and continuing through Data Release 1 (DR1), Rubin plans to begin nightly releases of single-epoch Processed Visit Images (PVIs) together with the associated Source catalogs of detections. 
+The PVIs will be subject to the standard 80-hour embargo period, while the catalogs are not but, for convenience and consistency, will be released together with the images after the 80-hour period following shutter close. 
```

Comment:
```suggestion
The PVIs will be subject to the standard 80-hour embargo period, while the catalogs are not but, for convenience and consistency, will be released together with the images after the 80-hour embargo.
```

I wouldn't reference shutter close here because there is ongoing discussion of whether the images drop at shutter + 80 or if we release each night in one batch.